lib/pages/laws.dart

```diff
@@ -31,15 +31,115 @@ class Laws extends StatelessWidget {
               children: <Widget>[
                 ListTile(
                   leading: Icon(Icons.label),
-                  title: Text('Law 1'),
+                  title: ListView(
+                    shrinkWrap: true,
+                    children: <Widget>[
+                      ListTile(
+                        title: Text('Family Violence Act:'),
+                      ),
+                      ListTile(
+                        title: Text('The law protects against physical, sexual, '
+                            + 'and emotional abuse among family members.'),
+                      ),
+                      ListTile(
+                        title: Text('Family members in this situation of violence '
+                            + 'can be defined as present or past spouses, parents '
+                            + 'of the same child, parents and children, stepparents '
+                            + 'and stepchildren, foster parents and foster children, '
+                            + 'or other persons living or formerly living in the '
+                            + 'same household.'),
+                      ),
+                      ListTile(
+                        title: Text('Violation of protective orders can results '
+                            + 'in a misdemeanor charge with up to 12 months in '
+                            + 'jail and a \$1000 fine.'),
+                      ),
+                      ListTile(
+                        title: Text('Violation of protective orders can also be '
+                            + 'considered stalking and can be sentenced as '
+                            + 'either misdemeanor or felony.'),
+                      ),
+                    ],
+                  ),
                 ),
                 ListTile(
                   leading: Icon(Icons.label),
```

Comment:
Again, I would remove this icon as I think it takes up a lot of space without adding visual clarity.